examples/papers/dfn_transport/example2/pot.py

```diff
@@ -190,39 +139,17 @@ def main():
         title = ["max $\\theta$", grid_label]
         # Alessio
         for method in methods_alessio:
-            data = (
-                folder_in
-                + method
-                + "/"
-                + "Cmax_"
-                + grid[0]
-                + ".csv"
-            )
+            data = folder_in + method + "/" + "Cmax_" + grid[0] + ".csv"
```

Comment:
This looks to me like we run different versions of black. Can you please check your versions? I would like to understand better how this thing evolves.

Reply:
black, version 18.9b0